src/providers/touchid/touchid.ts

```diff
@@ -91,6 +91,8 @@ export class TouchIdProvider {
             );
             return resolve();
           } else this.logger.debug("Didn't authenticate!");
+        }, () => {
+          return reject();
```

Comment:
It seems like this line is unnecessary if the `androidFingerprintAuth` Promise were directly returned instead of wrapped in a `new Promise`. Also this approach will not pass the rejection error to the `.catch` block.